play/vite.config.ts

```diff
@@ -39,14 +48,34 @@ export default defineConfig(({ mode }) => {
                     }
                 },
             }),
+            Icons({ compiler: "svelte" }),
             legacy({
                 //targets: ['defaults', 'not IE 11', 'iOS > 14.3']
 
                 // Structured clone is needed for Safari < 15.4
                 polyfills: ["web.structured-clone"],
                 modernPolyfills: ["web.structured-clone"],
             }),
+            tsconfigPaths(),
         ],
+        test: {
+            environment: "jsdom",
+            globals: true,
+            coverage: {
+                all: true,
+                include: ["src/*.ts", "src/**/*.ts"],
+                exclude: ["src/i18n", "src/enum"],
+            },
+        },
+        optimizeDeps: {
+            include: ["olm"],
```

Comment:
Do we need "olm" with the Matrix Rust lib?

Reply:
Yes, it's necessary. Without olm, matrix js sdk will raised an error related to missing OLM... I think the matrix rust js lib is again using olm.